helm/prometheus-rules/templates/platform/atlas/alerting-rules/monitoring-pipeline.rules.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: monitoring-pipeline.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: monitoring-pipeline
+    rules:
+    - alert: MetricForwardingErrors
+      annotations:
+        description: '{{`Monitoring agent can''t communicate with Remote Storage API at {{ $labels.url }}.`}}'
+        opsrecipe: monitoring-pipeline/
+        dashboard: promRW001/prometheus-remote-write
+      expr: |-
+        rate(prometheus_remote_storage_samples_failed_total[10m]) > 0.1
+          or rate(prometheus_remote_storage_samples_total[10m]) == 0
+          or rate(prometheus_remote_storage_metadata_retried_total[10m]) > 0
+      for: 1h
+      labels:
+        area: platform
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: atlas
+        topic: observability
+    - alert: JobScrapingFailure
+      annotations:
+        dashboard: servicemonitors-details/servicemonitors-details
+        description: '{{`Monitoring agents for cluster {{$labels.installation}}/{{$labels.cluster_id}} has failed to scrape all targets in {{$labels.job}} job.`}}'
+        summary: Monitoring agent failed to scrape all targets in a job.
+        opsrecipe: monitoring-job-scraping-failure/
```

Comment:
Do we currently alert for any job scraping failing? I think we only alert for "critical" ones and we have a few jobs failing on some clusters and consider it "normal".
That may be something we want to change, but just setting a new alert like that may result in an avalanche of pages! :sweat_smile:

Reply:
Oh, just noticed it's "notify only", so probably good :+1:

Reply:
...and it's basically the `PrometheusJobScrapingFailure` that was in `prometheus.rules.yml`.
Yet, I suspect it currently pages on most clusters :cry:

Reply:
Yes it does :( but I explained in the changelog that I Only renamed this alert